packages/rubrik/data_stream/drives/manifest.yml

```
@@ -33,15 +39,34 @@ streams:
         type: bool
         multi: false
         default: false
-      - name: processors
+      - name: pageSize
+        type: integer
+        title: Page Size
+        multi: false
+        required: false
+        show_user: false
+        description: The number of results to return per page.
+      - name: filters
         type: yaml
-        title: Processors
+        title: Filters
         multi: false
         required: false
         show_user: false
-        description: >
-          Processors are used to reduce the number of fields in the exported event or to enhance the event with metadata. This executes in the agent before the logs are parsed. See [Processors](https://www.elastic.co/guide/en/fleet/current/elastic-agent-processor-configuration.html) for details.
-
-    template_path: httpjson.yml.hbs
-    title: Rubrik Node Drives
-    description: Collect Rubrik Node drives data via the RSC GraphQL API
+        default: "# filter:\n#   - field: \"location\"\n#     texts: \n#       - \"prod-lab.local\"\n"
+        description: >-
+          Specify filters for refining the data. Filters must be defined as an array of Filter objects. Refer to the [Rubrik API documentation](https://rubrikinc.github.io/rubrik-api-documentation/schema/reference/filter.doc.html) for valid fields.
+      - name: resource_timeout
+        type: text
+        title: Resource Timeout
+        description: Duration before declaring that the HTTP client connection has timed out. Valid time units are ns, us, ms, s, m, h. Default is "30"s.
```

```suggestion
        description: Duration before declaring that the HTTP client connection has timed out. Valid time units are ns, us, ms, s, m, h. Default is 60s.
```
I think we overlooked this. We need to change this for all data streams.